app/controllers/api/v1/credentials_controller.rb

```diff
@@ -3,7 +3,16 @@ class CredentialsController < ApiController
     respond_to :json
```

Comment:
If it's not blowing then there might be a bug in Rails - it's meant to blow up at class definition time :smile: